Folderbartop

PierreDmyy opened the pull request:
Add component bar of folders page with one text and one button "add".

PierreDmyy committed:
del stories init

PierreDmyy committed:
merge

PierreDmyy committed:
add profil on navbartop with props

PierreDmyy committed:
add component FolderBarTop

PierreDmyy committed:
fix PR

dadtmt commented on src/stories/index.js:
```diff
@@ -24,6 +18,13 @@ import NavbarTopSearch from "../components/NavbarTopSearch"
 import NavbarTopNotifBubble from "../components/NavbarTopNotifBubble"
+import FoldersBarTop from "../components/FoldersBarTop"
+
+const contentOfProfil = {
```
for consistency propsContentOfProfile

PierreDmyy committed:
fix PR add propsContentProfil

PierreDmyy committed:
ok

Merge branch 'folderbartop' of github.com:WildCodeSchool/reims-0218-gyvit into folderbartop
